build/yamls/base/conf/antrea-controller.conf

```diff
@@ -53,3 +53,7 @@ featureGates:
 # API groups. After adding the annotation, legacy CRDs can be deleted safely without impacting
 # new CRDs.
 #legacyCRDMirroring: true
+
+# Enable flexible IPAM mode for Antrea. This mode allows to assign subnets to deployments and
```

Comment:
Deployments
StatefulSets

We capitalize the K8s resource types.

Reply:
Thanks, done